### PR TITLE
Update Cake to 0.28.0.

### PR DESCRIPTION
**Pull request by patriksvensson**

### CODE REVIEW

File: src/Cake.Graph/Cake.Graph.csproj [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <EmbeddedResource Include="Content\cytoscapewyam.cshtml" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Cake.Core" version="0.26.0" />
+    <PackageReference Include="Cake.Core" version="0.28.0" />
```
Comment (nmbro) [inline src/Cake.Graph/Cake.Graph.csproj]: Shouldn't that be 
```
<PackageReference Include="Cake.Core" version="0.28.0" PrivateAssets="All" />
```

**Comment by gep13** [comment]
@patriksvensson I have rebased this onto the develop branch, and merged in here: https://github.com/cake-contrib/Cake.Graph/commit/3fdb0c73b2c886f02992b743eda996fb1be9b82c

@nmbro yes, I will take care of that now that this is merged.

**Comment by wozzo** [comment]
@patriksvensson  @nmbro  @gep13 Thanks for doing this all! Been v.busy recently

**Comment by gep13** [comment]
@wozzo you are very welcome.  Totally understand being busy.  This is one of the reasons to move an addin into the Cake-Contrib organisation, so that others can help out when people are busy.  Thanks again for all your hard work in getting this addin created in the first place!

**Pull request closed without merging**